libs-scala/executors/src/main/scala/com/daml/Executors.scala

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.executors
+
+import com.codahale.metrics.{InstrumentedExecutorService, MetricRegistry}
+import com.daml.metrics.ExecutorServiceMetrics
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutorService}
+import java.util.concurrent.{ExecutorService, ThreadFactory, Executors => JavaExecutors}
+
+object Executors {
```

Comment:
These factory methods are great.
However, for improving readability (and to avoid the name clash with `java.util.concurrent.Executors`) I'd suggest renaming this to `InstrumentedExecutors`.